packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -94,25 +87,39 @@ const ShowHide = (props) => {
 
   const theme = React.useContext(ThemeContext);
   const contentRef = React.useRef(null);
+  const [activeElement, setActiveElement] = React.useState(null);
+  const focusRefClassName = 'show-hide-ref-focusable';
 
   /**
-   * Upon showing hidden content, set focus to the child element if focusRef provided, othervise set focus to the content container.
-   * That would allow assistive technologies to read the newly revealed content.
+   * Upon showing hidden content, set activeElement to focusRef element if provided, othervise set it to content container.
    */
   React.useEffect(() => {
+    let newActiveElement = null;
     if (isOpen) {
-      let element = null;
       if (focusRef?.current) {
-        element = focusRef?.current;
+        if (checkIfAddTabindex(focusRef?.current)) {
+          // not focusable element with no tabindex set, safe to temporary add tabindex that will later be removed on blur
+          focusRef?.current.setAttribute('tabIndex', '-1');
+          focusRef?.current.setAttribute('role', 'group');
```

Comment:
Do we want to change the role of the focus ref component?  Couldn't that have negative accessibility impacts.  For example, what if the ref was a table or some other role?  We would now be blindly making it a group role.

Reply:
Makes sense, I'll check the role first and re-assign it only in case there was no role. Will add a commit per that and update you once it done.

Reply:
Added in this commit: https://github.com/cerner/terra-core/pull/3838/commits/170f7cbd0b5ff79272fb22d69c3b7d268398c352